## <center>Практическое задание к лекции 5

___

**1. Известно, что генеральная совокупность распределена нормально, со средним квадратическим отклонением, $\displaystyle\sigma=16$. Найти доверительный интервал для оценки математического ожидания с надежностью $0.95$, если выборочная средняя $M=80$, а объем выборки $n=256$.**

Среднее квадратичное отклонение генеральной совокупности $\sigma=16$, отсюда - для расчета $95\$-го доверительного интервала, используем $Z$-критерий. Расчет по формуле:

$\displaystyle\bar X\pm z_{\alpha/2}\cdot\frac{\sigma}{\sqrt n}$, где:

$\bar X = M = 80$ - среднее значение выборки

$z_{alpha/2}$ - табличное значение $Z$-критерия

$z_{alpha/2} = 1,96$ - значение для %95\%-го  доверительного интервала

$\sigma = 16$ - среднее квадратичное отклонение генеральной совокупности

$n = 256$ - обьем выборки

Таким образом $95\%$-й доверительный интервал расчитаем как: 

$\left[80-1.96\cdot\frac{16}{\sqrt 256};80+1.96\cdot\frac{16}{\sqrt 256}\right]$ 

In [2]:
left = 80 - 1.96 * 16/256**(1/2)
right = 80 + 1.96 * 16/256**(1/2)
print(left, right)

78.04 81.96


---  
#### Ответ:  
$95\%$-й доверительный интервал для оценки мат. ожидания генеральной совокупности: $\left[78.04;81.96\right]$  

---

**2. В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью, получены опытные данные: 6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1 Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей, оценить истинное значение величины X при помощи доверительного интервала, покрывающего это значение с доверительной вероятностью 0,95.**

Поскольку из условия задачи нам ничего не известно ни о мат. ожидании генеральной совокупности ни о среднем квадратическом отклонении для генеральной совокупности, то для расчета $95\%$-го доверительного интервала будем использовать $t$-критерий,  и формулу:  

$\displaystyle\bar X\pm t_{\alpha/2}\cdot\frac{\sigma_{несмещ}}{\sqrt{n}}$, где  

$\sigma_{несмещ}$-несмещённое среднее квадратическое отклонение выборки,  

$n$-размер выборки  

Воспользуемся встроенными функциями библиотеки $numpy$:

In [3]:
import numpy as np

In [12]:
arr = np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])
np.mean(arr)
n = len(arr)
sigma = np.std(arr, ddof=1)
print(f'sigma = {round(sigma, 2)}')

sigma = 0.45


Вычисляем $t$-критерий:

In [13]:
import scipy.stats as stats

In [14]:
def t_from_table(confidens, len_array):
    alpha = (1 - confidens)
    return stats.t.ppf(1-alpha/2, len_array - 1)

In [15]:
def confidens_int(arr, confidens):
    return round(np.mean(arr)-t_from_table(confidens,len(arr))*np.std(arr, ddof=1)/len(arr)**0.5,3), \
           round(np.mean(arr)+t_from_table(confidens,len(arr))*np.std(arr, ddof=1)/len(arr)**0.5,3)

print(f'95%-й доверительный интервал для истинного значения Х: {confidens_int(arr, 0.95)}.')

95%-й доверительный интервал для истинного значения Х: (6.268, 6.912).


---  
#### Ответ:  

95%-й доверительный интервал для истинного значения Х: $\left[6.268;6.912\right]$  

---

**3. Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм. Используя односторонний критерий с $\alpha=0.05$ проверить эту гипотезу, если в выборке из $n=100$ шариков средний диаметр оказался равным 17.5 мм, а дисперсия известна и равна 4 кв.мм.**  

Сформулируем нулевую гипотезу: 


$H0: \mu=\mu_0=17$, где $\mu$ и $\mu_0$ средние выборки и генеральной совокупности и выборки, соответственно.  

Альтернативная гипотеза:  

$H1: \mu>\mu_0$  

Поскольку нам известны и среднее и дисперсия генеральной совокупности, то для проверки гипотезы будем использовать 
$z$-критерий.  

$z_н=\frac{\bar X-\mu_0}{\sigma/\sqrt n}$, где  

$\bar X=\mu=17.5$ среднее по выборке,  

$\sigma=\sqrt 4=2$ среднее квадратическое отклонение генеральной совокупности,  

$n=100$ размер выборки.  

In [18]:
z = (17.5-17)/0.2
print(f'Наблюдаемое значение z-критерия: {z: .1f}')

Наблюдаемое значение z-критерия:  2.5


In [19]:
zt = stats.norm.ppf(0.95)
print(f'Табличное значение z-критерия: {zt: .4f}')

Табличное значение z-критерия:  1.6449


Поскольку $z_н>z_t$, **отвергаем** нулевую гипотезу на уровне значимости $\alpha=0.05$

---  

#### Ответ:  

Отвергаем нулевую гипотезу $H_0: \mu_0=17$ на уровне значимости $\alpha=0.05$  

---